tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.html

```diff
@@ -0,0 +1,30 @@
+<div id="editor">
+	<p>Foo</p>
+</div>
+
+<script>
+
+	if ( bender.tools.env.mobile ) {
+		bender.ignore();
+	}
+
+	CKEDITOR.replace( 'editor', {
+		height: 300
+	} );
+</script>
+
+<button onclick="start()" >Start.</button>
+<script>
+	function start() {
```

Comment:
`Button` get focus after pressing it.
What more I cannot just use `ed.focus()` because in such case selection automatically shows caret at the beginning of line.
I cannot also click into editor to restore focus in editor, because such scenario also show caret.
Idea is to remove ranges without loosing focus on editor.

What just came on my mind is to create some div 20px x 20px, which might simulate similar case with `mouseenter` event. Because `click` event mess up with focus I need to avoid them in this case. What do you think about that?